java/code/src/com/redhat/rhn/domain/channel/Channel.hbm.xml

```diff
@@ -145,6 +145,41 @@ PUBLIC "-//Hibernate/Hibernate Mapping DTD 3.0//EN"
         -->
     </class>
 
+    <sql-query name="Channel.findAllByUserOrderByChild">
+        <![CDATA[
+    SELECT {c.*}
+    FROM rhnChannel {c}
+        LEFT OUTER JOIN rhnChannel parent ON c.parent_channel = parent.id
+        LEFT OUTER JOIN rhnChannelCloned c_1_ ON c.id = c_1_.id
```

Comment:
Nitpick: please use legible names for aliases :wink:

Reply:
@moio do you mean replace `c` with something like `channel`? or are um referring to the `<alias>_1_` part?

Reply:
I've now renamed the `c` alias to `channel`.